supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-single-sign-on.adoc

```diff
@@ -0,0 +1,146 @@
+[[red-hat-single-sign-on-conventions]]
+
+[discrete]
+[[access-token]]
+== Access token
+A token that can be provided as part of an HTTP request that grants access to the service on which the request is being invoked. This is part of the OpenID Connect and OAuth 2.0 specification.
+
+[discrete]
+[[assertion]]
+== Assertion
+Information about a user. This usually pertains to an XML blob that is included in a SAML authentication response that provided identity metadata about an authenticated user.
+
+[discrete]
+[[authentication]]
+== Authentication
+The process of identifying and validating a user.
+
+[discrete]
+[[authentication-flows]]
+== Authentication flows
+Authentication flows are work flows a user must perform when interacting with certain aspects of the system. A login flow can define what credential types are required. A registration flow defines what profile information a user must enter and whether something like reCAPTCHA must be used to filter out bots. Credential reset flow defines what actions a user must do before they can reset their password.
+
+[discrete]
+[[authorization]]
+== Authorization
+The process of granting access to a user.
+
+[discrete]
+[[client-adapters]]
+== Client adapters
+Client adapters are plugins that you install into your application environment to be able to communicate and be secured by Red Hat Single Sign-On. Red Hat Single Sign-On has a number of adapters for different platforms that you can download. There are also third-party adapters you can use for environments that Red Hat do not cover.
+
+
+[discrete]
+[[client-role]]
+== Client role
+Clients can define roles that are specific to them. This is basically a role namespace dedicated to the client.
+
+[discrete]
+[[client-scopes]]
+== Client scopes
+When a client is registered, you must define protocol mappers and role scope mappings for that client. It is often useful to store a client scope, to make creating new clients easier by sharing some common settings. This is also useful for requesting some claims or roles to be conditionally based on the value of `scope` parameter. Red Hat Single Sign-On provides the concept of a client scope for this.
+
+[discrete]
+[[clients]]
+== Clients
+Clients are entities that can request Red Hat Single Sign-On to authenticate a user. Most often, clients are applications and services that want to use Red Hat Single Sign-On to secure themselves and provide a single sign-on solution. Clients can also be entities that only want to request identity information or an access token so that they can securely invoke other services on the network that are secured by Red Hat Single Sign-On.
```

Comment:
```suggestion
Clients are entities that request Red Hat Single Sign-On to authenticate a user. Most often, clients are applications and services that want to use Red Hat Single Sign-On to secure themselves and provide a single sign-on solution. Clients are also entities that request identity information or an access token, so that they can securely invoke other services on the network that are secured by Red Hat Single Sign-On.
```